meinberlin/templates/a4categories/includes/module_categories_form.html

```diff
@@ -1,6 +1,9 @@
 {% load i18n %}
 
-<p class="form-hint">{% blocktranslate %}If you define categories, participants must assign their ideas accordingly (single choice, mandatory). Categories automatically appear in the display of ideas. In addition, the list of all ideas can be filtered by category.{% endblocktranslate %}</p>
+<p class="form-hint">{% blocktranslate %}testIf you define categories, participants must assign their ideas accordingly (single choice, mandatory). Categories automatically appear in the display of ideas. In addition, the list of all ideas can be filtered by category.{% endblocktranslate %}</p>
```

Comment:
@fuzzylogic2000 are you working off this branch? if so can you remove the 'test' before the 'if' sorry!